api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -436,6 +436,15 @@ enum ProductType {
   PRODUCT_TYPE_EUB = 2;
 }
 
+// SupportType if the type of support offered.
+enum SupportType {
+  SUPPORT_TYPE_UNKNOWN = 0;
```

Comment:
nit: _UNSPECIFIED instead of _UNKNOWN

```suggestion
  SUPPORT_TYPE_UNSPECIFIED = 0;
```

Reply:
Thanks, updated. I was following `ProductType`:

```
enum ProductType {
  PRODUCT_TYPE_UNKNOWN = 0;
```

But indeed `UNSPECIFIED` is used most of the time.

Reply:
Yeah that one is not "standard" too. A bunch of linters are disabled in the legacy/ folder because there were too many "offenders" when we enabled those, so we have to spot this kind of thing manually (and it slips sometimes).

Reply:
Good to know, thank you